interface/orders/procedure_provider_list.php

```diff
@@ -57,13 +57,13 @@ function refreshme() {
 // Process click to pop up the add window.
 function doedclick_add() {
  top.restoreSession();
- dlgopen('procedure_provider_edit.php?ppid=0', '_blank', 800, 750);
+ dlgopen('procedure_provider_edit.php?ppid=0', '_blank', 800, 750, false, '&nbsp;&nbsp;&nbsp;<i class="fa fa-plus"  aria-hidden="true"></i>&nbsp;' + '<?php echo xlt("Add Mode"); ?>&nbsp;&nbsp;&nbsp;');
 }
 
 // Process click to pop up the edit window.
 function doedclick_edit(ppid) {
  top.restoreSession();
- dlgopen('procedure_provider_edit.php?ppid=' + ppid, '_blank', 800, 750);
+ dlgopen('procedure_provider_edit.php?ppid=' + ppid, '_blank', 800, 750, false, '&nbsp;&nbsp;&nbsp;<i class="fa fa-pencil"  aria-hidden="true"></i>&nbsp;' + '<?php echo xlt("Edit Mode"); ?>&nbsp;&nbsp;&nbsp;');
```

Comment:
Just seems wrong to style bootstrap in this way. Set the title to a var instead of building in function parameter.

Reply:
I  can set it to a var , unfortunately had to use  nbsp; used for rtl compatibility

Reply:
Done

Reply:
Is this an issue for all icons with RTL? If so rec. adding the styling to the RTL style sheet or if an issue with dialogs(dlgopen is rtl aware) then you might add an icon option to dialog options list and let dlgopen style title. There is a global variable available for RTL detection from javascript in both frames and tabs defined as top.jsLanguageDirection = rtl/ltr etc. Problem with using fixed spaces are several(except maybe tables) in html5. Really should use css as I foresee an issue with this down the road and a global solution in css is best attacked now.